src/foam/nanos/controller/ApplicationController.js

```diff
@@ -423,6 +425,25 @@ foam.CLASS({
       class: 'Boolean',
       name: 'initSubject'
     },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.nanos.auth.login.ClientLoginService',
+      name: 'loginService',
+      factory: function() {
+        return foam.nanos.auth.login.ClientLoginService.create({}, this.__subContext__);
```

Comment:
This can create itself when the client loads if you add it to services jrl

Reply:
Look at cssTokenOverrideService

Reply:
updated, thanks @sarthak-marwaha